config/crd/bases/logging.openshift.io_clusterlogforwarders.yaml

```diff
@@ -114,6 +114,101 @@ spec:
                     name:
                       description: Name used to refer to the input of a `pipeline`.
                       type: string
+                    source:
+                      description: Source to receive logs from non-cluster sources.
+                      properties:
+                        http:
+                          description: HTTPSource receives encoded logs as a HTTP
+                            endpoint.
+                          properties:
+                            format:
+                              description: Format is the format of incoming log data.
+                              enum:
+                              - k8s
```

Comment:
How to specify this value?    the incoming data format may be different. For example: infrastructure logs include container logs and journal logs. which may result in different format?

Reply:
+1 - current preview is audit only for internal reasons. Other types  must be added in a later patch.